src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -14,14 +20,36 @@
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all assignments whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all assignments"
+            + " by its NAME, MODULE CODE, DEADLINE or PRIORITY. "
+            + "Finding is done one field at a time.\n"
+            + "DEADLINE keywords are in the format dd-MM-yyyy or HHmm, which allows finding of assignments by time "
+            + "and date separately.\n"
+            + "Parameters:" + PREFIX_NAME + "NAME [MORE NAMES] or\n"
+            + PREFIX_MODULE_CODE + "MODULE_CODE [MORE MODULE_CODES] or\n"
+            + PREFIX_DEADLINE + " DATE_OR_TIME_OF_DEADLINE [MORE DATE_OR_TIME_OF_DEADLINE] or\n"
+            + PREFIX_PRIORITY + " PRIORITY [MORE PRIORITIES]\n"
+            + "Example: " + COMMAND_WORD + " d/1200 24-10-2020 25-10-2020\n"
+            + "The example above finds all assignments due on 24 October 2020 and 25 October 2020 and finds all "
+            + "assignments due at 1200";
```

Comment:
I think the phrasing here is a little confusing. Could either mean assignments due at 1200 on the two dates, 24 Oct and 25 Oct, or it could mean all assignments due on 24 Oct and 25 Oct, and all assignments due at 1200 regardless of date. But maybe it's just me :P Just nitpicking

Reply:
Amended to make it sound a tad better LOL